src/app/model/movie/movie.ts

```diff
@@ -0,0 +1,129 @@
+export class Movie {
+  private _id: number;
+  private _title: string;
+  private _tagline: string;
+  private _vote_average: number;
+  private _vote_count: number;
+  private _release_date: Date;
+  private _poster_path: string;
+  private _overview: string;
+  private _budget: number;
+  private _revenue: number;
+  private _genres: string[];
+  private _runtime: number;
+
+
+  constructor(id: number, title: string, tagline: string, vote_average: number, vote_count: number, release_date: Date, poster_path: string, overview: string, budget: number, revenue: number, genres: string[], runtme: number) {
+    this._id = id;
+    this._title = title;
+    this._tagline = tagline;
+    this._vote_average = vote_average;
+    this._vote_count = vote_count;
+    this._release_date = release_date;
+    this._poster_path = poster_path;
+    this._overview = overview;
+    this._budget = budget;
+    this._revenue = revenue;
+    this._genres = genres;
+    this._runtime = runtme;
+  }
+
+
+
+
+  get id(): number {
```

Comment:
this whole range with get and set methods doesn't do anything useful, it can be all removed from Movie class. The private props can be marked as public and it'll work the same